lib/renderer/callbacks-registry.ts

```diff
@@ -23,14 +24,14 @@ class CallbacksRegistry {
     let filenameAndLine
     let match
 
-    while ((match = regexp.exec(stackString)) !== null) {
+    while ((match = regexp.exec(stackString!)) !== null) {
       const location = match[1]
       if (location.includes('(native)')) continue
       if (location.includes('(<anonymous>)')) continue
       if (location.includes('electron/js2c')) continue
 
       const ref = /([^/^)]*)\)?$/gi.exec(location)
-      filenameAndLine = ref[1]
+      filenameAndLine = ref![1]
```

Comment:
I'd prefer a null check here instead of telling TS we know better, the stack trace here could be Bad somehow (user overrides `global.Error` or something idk